### ISSUE TITLE
Update the description of an existing test remove the configured Variables & Secrets

### ISSUE THREAD

**Issue by sylock** [kubeshop/testkube#4440]
**Describe the bug**
When a test was already created. You modify the description and click on "save". Then you loose the Variables & Secrets.

**To Reproduce**
1. Create new test
- Fill in name
- Type: curl/test
- Source: String and just put "{}"
- No label
2. Go to Settings / Variables & Secrets on the test and add a var
- E.g. MYVAR value "toto"
- Then save
3. Go to Settings / Test on the test and add a description
- E.g. "my description" 
- Then save
4. Check vars again -> it's now empty

**Expected behavior**
Updating the field description must not modify other test configuration.

**Version / Cluster**
- API Version v1.14.5
- Dashboard Version v1.14.2

**Comment by vsukhin** [comment]
thank you @sylock our FrontEnd team will check it out

**Comment by smals-vinve** [comment]
This bug is still present and actually reproducible on https://demo.testkube.io

I've added a test which resulted in the following definition:
```yaml
apiVersion: tests.testkube.io/v3
kind: Test
metadata:
  name: bug-4440
  namespace: testkube
  labels:
    executor: curl-executor
    test-type: curl-test
spec:
  type: curl/test
  content:
    type: string
    repository:
    data: "{}"
  executionRequest:
    variables:
      MY_VAR:
        name: MY_VAR
        value: "somevalue"
        type: basic
    executePostRunScriptBeforeScraping: false
```

Then, I add a description via General / Description and save again. The final result is like this, with some fields lost while patching the description field in:

```yaml
apiVersion: tests.testkube.io/v3
kind: Test
metadata:
  name: bug-4440
  namespace: testkube
  labels:
    executor: curl-executor
    test-type: curl-test
spec:
  description: "some description"
  type: curl/test
  content:
    type: string
    repository:
    data: "{}"
```

**Comment by rangoo94** [comment]
Hi @sylock and @smals-vinve!

We have just released the fix in **Testkube Dashboard v1.16.2** (Helm chart: v1.16.13). 🎉 

Once again, sorry for the inconvenience!
